parser.py

```diff
@@ -452,3 +379,6 @@ def real(self, args):
 
     def nninteger(self, n):
         return int(unpack(n))
+
+    def parameter(self, name):
+        return sympy.Symbol(name[0])
```

Comment:
To deal with symbolic parameters in expressions

Reply:
From what I could see, you use this as the synonym for `id`. However, in qasm, `id` can refer to other kinds of identifiers, such as gate names, which I suspect we wouldn't want to be sympy symbols